### PR TITLE
depgraph.py: fix wrong type annotations

### PR DESCRIPTION
**Pull request by berinaniesh**
bug: https://bugs.gentoo.org/910171

### CODE REVIEW

**Comment by thesamesam** [comment]
cc @xxc3nsoredxx

File: lib/_emerge/depgraph.py [hunk 1]
```diff
@@ -11536,7 +11540,7 @@ def resume_depgraph(
     settings: portage.package.ebuild.config.config,
     trees: portage._trees_dict,
     mtimedb: Any,
-    myopts: Dict[str, str],
+    myopts: Dict[str, Union[str, int, bool]],
     myparams: Dict[str, Any],
```
Comment (xxc3nsoredxx) [inline lib/_emerge/depgraph.py:11544]: While we're at it...
```suggestion
    myparams: Dict[str, Union[int, str, bool]],
```
based on lines 11417 and 11439?

Am I correct in assuming the `Dict[str, Any]` was meant as a placeholder while figuring out a better/more exact type, but you forgot to replace them here?

File: lib/_emerge/depgraph.py [hunk 2]
```diff
@@ -11554,7 +11558,7 @@ def _resume_depgraph(
...
```
Comment (xxc3nsoredxx) [inline lib/_emerge/depgraph.py:11562]: Ditto.

**Review by xxc3nsoredxx — CHANGES_REQUESTED** [review]

File: lib/_emerge/depgraph.py (hunk 1 above)
Comment (berinaniesh) [reply lib/_emerge/depgraph.py:11544]: This file was my first ever type annotation. I had no clue on what's going on. I'll check the whole file once again @xxc3nsoredxx. Having said that,
```py
myparams = {
    "recurse": True,
    "binpkg_respect_use": "auto",
    "autounmask": True,
    "autounmask_keep_use": False,
    "autounmask_keep_license": True,
    "autounmask_keep_keywords": True,
    "autounmask_keep_masks": True,
    "bdeps": "auto",
    "ignore_soname_deps": "y",
    "dynamic_deps": True,
    "implicit_system_deps": True,
    "selective": True,
    "deep": True,
    "binpkg_changed_deps": "auto",
}

myopts = {
    "--newuse": True,
    "--pretend": True,
    "--update": True,
    "--deep": True,
    "--keep-going": True,
    "--quiet": True,
    "--regex-search-auto": "y",
    "--verbose": True,
}
```

**Pull request closed without merging**